README.md

```diff
@@ -38,6 +38,30 @@ To test the installation, run:
 zkpig version
 ```
 
+### Build from Source
+
+To build ZK-PIG from source, you'll need:
+- Go 1.21 or later
+- Git
+- Make
+
+1. Clone the repository:
+
+```sh
+git clone https://github.com/kkrt-labs/zk-pig.git
+cd zk-pig
+```
+
+2. Build the binaries:
+
+```sh
+make goreleaser-snapshot
```

Comment:
I think you're doing something too complicated. 

Just `go build` at the project root would be enough. 
Please test locally in your IDE before posting a PR.